packages/react-components/.babelrc.json

```diff
@@ -1,3 +0,0 @@
-{
-  "plugins": ["module:@fluentui/babel-make-styles", "annotate-pure-calls", "@babel/transform-react-pure-annotations"]
```

Comment:
I'd leave this as is for reason mentioned in other comment

Reply:
I restored this, but as Griffel packages (see other comment).